### PR TITLE
removing need for users to dismiss toast popup to save hacks

### PR DESCRIPTION
**Pull request by ArcerionDev**
used mass replace because I didn't want to sort through all 60 instances of await Toast.fire, going to test from this branch and see if it messes anything up, feel free to also look through and check if I screwed everything up

### CODE REVIEW

File: willsCheatMenu/src/hacks/inventory.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ const itemify = (item: Item[], amount: number) =>
 new Hack(category.inventory, "Item stacker").setClick(async () => {
 	const num = await NumberInput.fire("Amount", "How many of every item would you like?", "question");
 	if (!num.value) return;
-	if (!(await Confirm.fire("Are you sure you want to get all items in the game?"))) return;
+	if (!(await Confirm.fire("Are you sure you want to get all items in the game?")).value) return;
```
Comment (ArcerionDev) [inline willsCheatMenu/src/hacks/inventory.ts:21]: also made a inconsequential thing that keeps the item stacker from triggering even if the user denies the conf

Comment (ArcerionDev) [reply willsCheatMenu/src/hacks/inventory.ts:21]: cus the item stacker was triggering even if you cancel the confirmation

**Comment by afkvido** [comment]
YESSSSSSSSSSSSSSSSSSSS

**Review by 32n — APPROVED** [review]
Works on my computer.

**Comment by 32n** [comment]
Sorry, just updating some package-lock stuff. I probably should've done that in master.

**Comment by ArcerionDev** [comment]
all good, if it lgty then i'll merge

**Review by donaldli2020 — COMMENTED** [review]
cool

**Pull request merged**